docs/en/start-v2/kubernetes/kubernetes.mdx

```diff
@@ -36,6 +36,8 @@ To run the image with SeaTunnel, first create a `Dockerfile`:
   defaultValue="flink"
   values={[
     {label: 'Flink', value: 'flink'},
+    {label: 'zeta (local-mode)', value: 'zeta (local-mode)'},
```

Comment:
Would it be better to change the label to 'Zeta'

Reply:
I'll adjust it

Reply:
done